components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/devportal-api.yaml

```diff
@@ -5151,6 +5151,12 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/KeyManagerApplicationConfiguration'
+        alias:
+          type: string
+          description: |
+            The alias of Identity Provider.
+            If the tokenType is EXCHANGED, the alias value should be equal to the audience value of the JWT token
+          example: https://localhost:8243/token
```

Comment:
Shall we use the Resident KM issuer as example here?

Reply:
Fixed via 446173df7a595cd00744cb964c8a6f7597b5d274